middleware/2023-05-17-1.3.46.md

```diff
@@ -0,0 +1,53 @@
+---
+authors: poscreator
+slug: middleware/1.3.46
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.46 (Germany)
+In this version of the Middleware we solved an issue where DSFinV-K exports failed, we have updated the Fiskal Cloud Connector (FCC) to version 4.0.5 and we updated the CertificationId to the newest one. We also updated the mono launcher server packages as well as the install-service.sh script
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Updated third-party dependencies: Deutsche Fiskal and SwissbitCloud
+We've updated the FCC to version 4.0.5
+### Affected packages:
+- _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal  v1.3.46_
+- _fiskaltrust.Middleware.SCU.DE.SwissbitCloud v1.3.46_
+
+## Bug Fix: Certificate display for Deutsche Fiskal and SwissbitCloud
+We have updated the CertificationId to the newest one for Deutsche Fiskal and SwissbitCloud.
+### Affected packages:
+- _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal  v1.3.46_
+- _fiskaltrust.Middleware.SCU.DE.SwissbitCloud v1.3.46_
+
+## Bug Fix: DSFINV-K export failed
+We have fixed a bug in the DSFinV-K export which led to the export to crash for a very specific case when data was missing. We have also improved export performance for EF Queues and added a parameter to disable references in the export, which can be helpful for getting exports with wrongly configured implementations. 
+### Affected packages:
+- _fiskaltrust.Middleware.Queue.EF v1.3.46_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.46_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.46_
+
+## Updated mono launcher
+We've updated the mono launcher server packages as well as the install-service.sh script.
+
+# Middleware 1.3.46 (Italy)
+In this version of the Middleware we created the first version of the italian queue and SCU. We also implemented the first operations such as :
```

Comment:
should we add additional businesscases? e.g.:
- pos-receipt
- refund
- void
- out of operation

and maybe point to the updated postman collection?